prow/gangway/example/main.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
```

Comment:
this file might need to be moved to be under `prow -> cmd`. 

This repo doesn't really strictly follow the Go project layout https://github.com/golang-standards/project-layout, aka `cmd` directory for main.go and `pkg` directory for all core logics/helpers. In this repo cmds are under `prow/cmd`, and all other directories under `prow` are considered pkgs.

I would suggest move this file to be under `prow/cmd/gangway/example` if that makes sense to you

Reply:
This binary is **not** meant to be run in the service cluster next to tide, sinker, etc. Putting it in the same `cmd` subdir would be misleading IMHO.

Reply:
agree that it doesn't fit into `cmd` that well. However I still don't think it fits in a directory under "pkg" like directory in this repo. Looking at the `project-layout` project more and https://github.com/golang-standards/project-layout#examples seems make sense. Since the `cmd` dir for Prow is under `prow/cmd`, how about `prow/examples/gangway`?

Reply:
`prow/examples/gangway` is a better location indeed; done.